Archived 2022

SrijitaSarkar99 opened the pull request:
Changes done -
1. Updated timeline for the year 2023
2. Archived 2022 data
3. Added statistics of the year 2022

BenzeneAlcohol commented:
Update the global assets as well.

Try to remove the commented out code wherever they are, right now there are lot of places where there is commented out code, just remove them whenever you encounter them.

SrijitaSarkar99 commented:
Changes done:
1. Added projects and organizations of 2022 to global assets
2. Removed commented out codes from files in archivedorganizations and archivedprojects folders

kavyabhat02 commented:
1. Guidelines section - update from 2022 -> 2023 where required.
2. It would be better to have different routes to the organizations and their projects according to the year. Example: For COSH NITK's archived projects for 2022: The route could be ../2022/organizations/COSH_NITK instead of ../organizations/COSH_NITK
